dsc/decisionMaker.md

```diff
@@ -0,0 +1,36 @@
+# Desired State Configuration Overview for Decision Makers #
+
+This document describes the business benefits of using PowerShell Desired State Configuration (DSC). It is not a technical guide.
+
+## What Is Desired State Configuration? ##
+
+Windows PowerShell Desired State Configuration (DSC) provides a configuration platform built into Windows that is based on open standards. DSC is flexible enough to function reliably and consistently in each stage of the deployment lifecycle (development, test, pre-production, production), as well as during scale-out. 
```

Comment:
Is this okay? 

"PowerShell Desired State Configuration (DSC) is a configuration management platform built into Windows that is based on open standards"